CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ if(NAPI_JAVASCRIPT_ENGINE STREQUAL "V8" AND JSRUNTIMEHOST_CORE_APPRUNTIME_V8_INS
     set_property(TARGET llhttp_static PROPERTY FOLDER Dependencies)
 endif()
 
-if(JSRUNTIMEHOST_TESTS)
+if(JSRUNTIMEHOST_TESTS AND (WIN32 OR (APPLE AND NOT IOS) OR (UNIX AND NOT ANDROID)))
```

Comment:
Shouldn't this be the same condition as line 136? Can we move this to the Tests folder completely?